presto-orc/src/test/java/com/facebook/presto/orc/TestingColumnHandle.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Why new class if there's an identical `TestingColumnHandle` in presto-spi already?
When you get rid of it then you can remove https://github.com/prestodb/presto/pull/5998/files#diff-782082843d4225dee861d4934e26bdc4R91.